hphp/runtime/vm/jit/vasm-dead.cpp

```diff
@@ -94,6 +95,8 @@ bool effectful(Vinstr& inst) {
     case Vinstr::fctidz:
     case Vinstr::fcvtzs:
     case Vinstr::imul:
+    case Vinstr::smulh:
+    case Vinstr::mul:
```

Comment:
I think everything else in this switch is sorted alphabetically; can you follow suit?